fdm-app/app/routes/farm.$b_id_farm.field.$b_id.soil._index.tsx

```
@@ -0,0 +1,228 @@
+            // Remove the analysis
+            await removeSoilAnalysis(fdm, session.principal_id, a_id)
+            return redirectWithSuccess("./", {
+                message: "Bodemanalyse is verwijderd! 🎉",
+            })
+        } else {
+            throw data("Method not allowed", {
+                status: 405,
+                statusText: "Method not allowed",
+            })
+        }
+    } catch (error) {
```

Biome (1.9.4)

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)